package.json

```diff
@@ -13,14 +13,27 @@
     "build:website": "yarn workspace @emoji-mart/website build",
     "prettier": "prettier --check .",
     "prettier:fix": "prettier --write .",
-    "test": "yarn node --experimental-vm-modules $(yarn bin jest)"
+    "test": "jest"
```

Comment:
Running `yarn test` from the root gives a bunch of:

```
ts-jest[config] (WARN) message TS151001: If you have issues related to imports, you should consider setting `esModuleInterop` to `true` in your TypeScript configuration file (usually `tsconfig.json`). See https://blogs.msdn.microsoft.com/typescript/2018/01/31/announcing-typescript-2-7/#easier-ecmascript-module-interoperability for more information.
```

Perhaps adding a `tsconfig.json` similar to `./packages/emoji-mart/tsconfig.json` to the root of the project is a good idea? I'd also suggest adding `"noEmit": true` to both, since they're just used for IDE checking & in Jest.